frontend/src/components/Table.tsx

```diff
@@ -50,7 +50,7 @@ const Table = <T,>({
   const sort = useTableColumnSort<T>(columns, 0);
 
   const showPagination = enablePagination && allData.length > minPageSize;
-  const pagination = (pageDirection: 'up' | 'down') => (
+  const pagination = (pageDirection: 'up' | 'down', variant: 'top' | 'bottom') => (
```

Comment:
Can we not manage these two as the same thing. Variant is actually better than `pageDirection`. This was a quick fix for me back when I was doing the original table stuff. I think we can just use variant. `top` variant clearly needs the page to go down. Naturally, `bottom` would want it to go up.

Reply:
Ah we certainly can! Great callout - updated.